pkgs/servers/matrix-registration/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, pkgs, fetchFromGitHub, python3Packages }:
+
+let
+  # officially supported database drivers
+  dbDrivers = with python3Packages; [
+    psycopg2
+    # sqlite driver is already shipped with python by default
+  ];
+
+in
+buildPythonPackage rec {
+  pname = "matrix-registration";
+  version = "1.0.0.dev3";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "ZerataX";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0dz60qv32dn0ci4c696c12rlqamd00m19rn8c6dxbza23r7a952n";
+  };
+
+  postPatch = ''
+    sed -i -e '/alembic>/d' setup.py
+  '';
+
+  propagatedBuildInputs = [
+    pkgs.libsndfile
```

Comment:
Probably sufficient for this to be a buildInput, can you check?

Reply:
oh that's just remnants from my `shell.nix`
I think I can just remove this